src/scrapers/game_details_scrape.py

```
@@ -22,9 +22,46 @@ def clean_name(name):
+def scrape_sidearm_story_recap(url):
+    """
+    Extract headline and published time from a Cornell Sidearm story/recap page
+    """
+    if not url:
+        return {}
+    try:
+        response = requests.get(url, headers=HTTP_REQUEST_HEADERS, timeout=20)
+        if response.status_code != 200:
+            return {}
+        soup = BeautifulSoup(response.text, "html.parser")
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)